app/services/feeds/basic.rb

```diff
@@ -0,0 +1,42 @@
+module Feeds
+  class Basic
+    def initialize(user: nil, number_of_articles: 25, page: 1, tag: nil)
+      @user = user
+      @number_of_articles = number_of_articles
+      @page = page
+      @tag = tag
+    end
+
+    def feed
+      articles = Article.published.order("hotness_score DESC")
+        .where("score >= 0")
```

Comment:
If we want to lean fully into the AR DSL:

```suggestion
        .where(score: 0..)
```